### PR TITLE
GH Actions

### PR DESCRIPTION
**Pull request by evgenyz**
Set up GH Actions CI.

Fixed Travis MacOS build.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -6,14 +6,14 @@ name: CI
...
+    branches: [ master, maint-1.3 ]
   pull_request:
-    branches: [ master ]
+    branches: [ master, maint-1.3 ]
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
   # This workflow contains a single job called "build"
-  build:
+  ubuntu:
```
Comment (jan-cerny) [inline .github/workflows/main.yml:16]: Would it be possible to specify specific versions of Ubuntu?

**Comment by evgenyz** [comment]
> It doesn't build the yamlfilecontent probe. Can you look at it?

Dammit, again :(

File: .github/workflows/main.yml (hunk 1 above)
Comment (evgenyz) [reply .github/workflows/main.yml:16]: It is possible, but maybe it would be great to follow the edge in our CI?

**Comment by jan-cerny** [comment]
> > It doesn't build the yamlfilecontent probe. Can you look at it?
> 
> Dammit, again :(

I can see that it uses --no-recurse-submodules.

File: .github/workflows/main.yml (hunk 1 above)
Comment (ggbecker) [reply .github/workflows/main.yml:16]: what about using matrix to run multiple jobs: https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

you can run the action on multiple versions at the same time

Comment (evgenyz) [reply .github/workflows/main.yml:16]: I'm not sure how hardly we could abuse GH builders, but okay. What do we want? ubuntu-latest + ubuntu 18.04 + ubuntu 20.04; macos-latest?

Comment (evgenyz) [reply .github/workflows/main.yml:16]: Options: https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#github-hosted-runners

Comment (evgenyz) [reply .github/workflows/main.yml:16]: Hold on a second. I forgot Windows :). I'll add it

Comment (jan-cerny) [reply .github/workflows/main.yml:16]: Windows are already covered by AppVeyor and old Ubuntu is already covered by Travis.

**Comment by evgenyz** [comment]
So, yaml probe is in place but it will fail until we fix dpkg probe in 20.04.

**Comment by evgenyz** [comment]
Okay, dpkginfo is now fixed. Let's have it.

**Comment by evgenyz** [comment]
Aha, now password probe test is failing on 20.04. Bummer!

**Comment by evgenyz** [comment]
Finally!

**Comment by jan-cerny** [comment]
Great, let's try it in action, we can always change it later.

**Pull request merged**